forge-gui/res/cardsfolder/g/genesis_of_the_daleks.txt

```diff
@@ -0,0 +1,15 @@
+Name:Genesis of the Daleks
+ManaCost:4 B B
+Types:Enchantment Saga
+K:Chapter:4:DBToken,DBToken,DBToken,DBVillainous
+SVar:DBToken:DB$ Token | TokenAmount$ X | TokenScript$ b_3_3_a_dalek_menace | SpellDescription$ Create a 3/3 black Dalek artifact creature token with menace for each lore counter on CARDNAME.
+SVar:DBVillainous:DB$ VillainousChoice | Choices$ DBDestroyDalek,DBDestroyNonDalek | ValidTgts$ Opponent | SpellDescription$ Target opponent faces a villainous choice — Destroy all Dalek creatures and each of your opponents loses life equal to the total power of Daleks that died this turn, or destroy all non-Dalek creatures.
+SVar:DBDestroyDalek:DB$ DestroyAll | ValidCards$ Creature.Dalek | RememberDestroyed$ True | SubAbility$ DBLoseLife | SpellDescription$ Destroy all Dalek creatures and each of your opponents loses life equal to the total power of Daleks that died this turn.
+SVar:DBLoseLife:DB$ LoseLife | LifeAmount$ TotalPower | Defined$ Opponent | SubAbility$ DBCleanup
```

Comment:
wrong amount

Reply:
Well the approach to get the LKI is there now
but the power part is missing

Reply:
Hmm` Count$SumPower_ThisTurnEntered_Graveyard_from_Battlefield_Creature.Dalek` isn't working. Any pointers ?

Reply:
We might need to add support if it's new

Reply:
How should it be scripted after your changes ?

Reply:
You should be able to chain it cleanly with handlePaid syntax now:
`Count$ThisTurnEntered_Graveyard_from_Battlefield_Dalek$CardPower`